src/eko/evolution_operator/__init__.py

```diff
@@ -579,6 +579,9 @@ def quad_ker_qed(
     return ker
 
 
+OPMEMBERS = Dict[Tuple[int, int], OpMember]
```

Comment:
This is a type hint, not a constant. The naming convention (i.e. the case) should be the same of classes (sorry to be pedantic...)

Reply:
https://github.com/NNPDF/eko/pull/291/commits/494da3c0ccd3d081cc3e345f39e96457b0fd46a3